freqtrade/exchange/binance.py

```diff
@@ -66,29 +107,111 @@ def stoploss(self, pair: str, amount: float, stop_price: float, order_types: Dic
 
             rate = self.price_to_precision(pair, rate)
 
-            order = self._api.create_order(symbol=pair, type=ordertype, side='sell',
-                                           amount=amount, price=rate, params=params)
+            order = self._api.create_order(
+                symbol=pair,
+                type=ordertype,
+                side=side,
+                amount=amount,
+                price=rate,
+                params=params,
+                leverage=leverage
```

Comment:
I don't think this is correct for binance.

Binance won't accept `leverage` as parameter - also not for stop orders.

Reply:
I think I just got this confused with the freqtrade create_order class

Reply:
so do i - and that's perfectly fine (this is why we do reviews) :laughing: